packages/flutter/lib/src/widgets/overflow_bar.dart

```diff
@@ -125,6 +126,23 @@ class OverflowBar extends MultiChildRenderObjectWidget {
   /// Defaults to 0.0.
   final double spacing;
 
+  /// Defines the [children]'s horizontal layout according to the same
+  /// rules as for [Row.mainAxisAlignment].
+  ///
+  /// If this property is non-null, and the [children] - separated by
+  /// [spacing] - fit within the available width, then the overflow
+  /// bar's will be as wide as possible. Otherwise it will be no
+  /// wider than needed to layout the [children] separated by
+  /// [spacing], modulo the incoming constraints.
+  ///
+  /// If [alignment] is one of [MainAxisAlignment.spaceAround],
+  /// [MainAxisAlignment.spaceBetween], or
+  /// [MainAxisAlignment.spaceEvenly], then the [spacing] parameter is
+  /// only used to see if the horizontal layout will overflow.
+  ///
```

Comment:
Do we need to describe what happens in the case of an alignment set and it does need to overflow?

Reply:
Good point. In that case overflowAlignment applies. This would be a good place to mention overflowAlignment, I'll add it.